### PR TITLE
Fix build failure

### PR DESCRIPTION
**Pull request by cathyzhyi**

### CODE REVIEW

**Review by ljfitz — APPROVED** [review]

**Comment by sjarus** [comment]
@cathyzhyi on a clean workspace containing this and #547 , I see all that the BatchNorm cases we just fixed are failing. The signature is prior to legalization: 

        Exception:
        Lowering TorchScript Object Graph IR -> Torch Backend IR failed with the following diagnostics:
        error: 'torch.prim.RaiseException' op expected 2 operands, but found 1
        note: see current operation: "torch.prim.RaiseException"(%9) : (!torch.str) -> ()
        error: 'torch.prim.RaiseException' op expected 2 operands, but found 1
        note: see current operation: "torch.prim.RaiseException"(%11) : (!torch.str) -> ()
        error: 'torch.prim.RaiseException' op expected 2 operands, but found 1
        note: see current operation: "torch.prim.RaiseException"(%9) : (!torch.str) -> ()
        error: 'torch.prim.RaiseException' op expected 2 operands, but found 1
        note: see current operation: "torch.prim.RaiseException"(%11) : (!torch.str) -> ()

Do you have any suggestions on how to fix this ?

**Comment by cathyzhyi** [comment]
@sjarus The latest PyTorch nightly build (1.11.0.dev20220128) start expecting 2 arguments. Probably you are using an older version of pytorch?

**Comment by sjarus** [comment]
Thanks @cathyzhyi it works with 3.9.10, which I picked up based on what the CI uses here. We'd previously been on 3.6.9, which doesn't get any new pytorch versions since 2021-end.
